test/Orchard.Tests/Extensions/TestDependencyModules/DerivedThemeSample2/Theme.txt

```diff
@@ -0,0 +1,5 @@
+﻿Name: Derived Theme Sample
+Author: Nicholas Mayne
+Description: Base Theme
+Version: 1.0.0 
+BaseThemes: BaseThemeSample, BaseThemeSample2
```

Comment:
BaseThemes and Dependencies are the same terms, there is no difference in a Base Theme/Theme/Module etc etc. From an extensions perspective, they are just extensions.

Themes use the BaseThemes property as a way to say, hey, I depend on you. You could put this in the Dependencies property, and semantically that would be fine.

Now... When I say they are all the same, they only holds true within the Extensions code. Within the Display code, there is a ThemeExtensionInfo that has other things. That I think uses BaseThemes for something.